app/models/access.rb

```diff
@@ -0,0 +1,15 @@
+class Access < ApplicationRecord
```

Comment:
We've stuck to the name `Access`. Because didn't like:

* Role (this does not represent a role, it's a composite)
* UserResource (is unwieldy)
* Region (Facilities and Organizations are not regions)